### PR TITLE
Perf: updates blacklist to exclude unused modules

### PR DESCRIPTION
**Pull request by ukilon-okta**
**Before:** 
```
$ ls -lh dist/js/
total 8864
-rw-r--r--  1 ukilon  staff   1.0M Oct 18 21:02 okta-sign-in-no-jquery.js
-rw-r--r--  1 ukilon  staff   2.2M Oct 18 21:02 okta-sign-in.entry.js
-rw-r--r--  1 ukilon  staff   1.1M Oct 18 21:02 okta-sign-in.min.js
```

**After:**
```
$ ls -lh dist/js/
total 7472
-rw-r--r--  1 ukilon  staff   861K Oct 18 21:00 okta-sign-in-no-jquery.js
-rw-r--r--  1 ukilon  staff   1.9M Oct 18 21:00 okta-sign-in.entry.js
-rw-r--r--  1 ukilon  staff   957K Oct 18 21:00 okta-sign-in.min.js
```

### CODE REVIEW

File: webpack.common.config.js [hunk 1]
```diff
@@ -24,7 +24,8 @@ module.exports = function (outputFilename) {
         'okta/underscore': SHARED_JS + '/util/underscore-wrapper',
         'okta/handlebars': 'handlebars/dist/handlebars',
         'okta/moment': 'moment/moment',
-        'okta/moment-tz': VENDOR + '/lib/moment-timezone-with-data-2010-2020',
+        'okta/moment-tz': EMPTY,
```
Comment (rchild-okta) [inline webpack.common.config.js]: @mauriciocastillosilva-okta quick search suggests we're not using this, but can you confirm?

File: webpack.common.config.js [hunk 2]
```diff
@@ -100,10 +86,34 @@ module.exports = function (outputFilename) {
...
+            'views/components/ConfirmationDialog',
+            'views/components/MultiViewModalDialog',
+            'views/components/DropDown',
+            'views/wizard/BaseWizard',
           ].map(function (file) {
             return path.resolve(TARGET_JS, 'shared', file);
-          })
+          }).concat([
+            /moment-tz/,
+            /vendor\/plugins\/vkbeautify/,
```
Comment (mauriciocastillosilva-okta) [inline webpack.common.config.js]: nit: do we need to escape the regular ```/``` ? Some below do not, and these ones here do ```\/```. Maybe we can have them all the same

File: webpack.common.config.js (hunk 1 above)
Comment (mauriciocastillosilva-okta) [reply webpack.common.config.js]: yeah, I don't see it either. And I can't think of a UI where we display times... errors like 'we sent you an sms in the last minute' or 'do not challenge me for N minutes on this device' are all not using moment as far as I know.

File: webpack.common.config.js (hunk 2 above)
Comment (ukilon-okta) [reply webpack.common.config.js]: this is a regex literal, if it was a string we wouldn't have to

Comment (mauriciocastillosilva-okta) [reply webpack.common.config.js]: doh.... missed that somehow
👍

**Comment by ukilon-okta** [comment]
@hor-kanchan-okta okta-core is 100% green. can you guys please merge?

**Comment by hor-kanchan-okta** [comment]
@ukilon-okta I can merge after you rebase. "This branch is out-of-date with the base branch"

**Comment by mauriciocastillosilva-okta** [comment]
what happened to the button  for "rebasing" in github? It used to be here... why did we lose that functionality?

**Comment by mauriciocastillosilva-okta** [comment]
oh... maybe it is because it is not a topic branch and it is @ukilon-okta's master?

**Pull request merged**